app/src/main/java/com/example/mysimplecoindeck/repository/CoinRepository.kt

```diff
@@ -1,12 +1,22 @@
 package com.example.mysimplecoindeck.repository
 
 import com.example.mysimplecoindeck.models.CoinsResponse
+import com.example.mysimplecoindeck.models.dbModels.CoinPortfolioEntity
+import com.example.mysimplecoindeck.models.searchSuggestions.SearchResponse
 import com.example.mysimplecoindeck.models.singleCoin.CoinResponse
 import kotlinx.coroutines.flow.Flow
 import retrofit2.Response
 
 interface CoinRepository {
     val coinsList: Flow<Response<CoinsResponse>>
 
-    fun coinDetail(uuid: String): Flow<Response<CoinResponse>>
+    suspend fun coinDetail(uuid: String): Flow<Response<CoinResponse>>
```

Comment:
`suspend` and return type `Flow<>` does not make sense, either a suspend function that returns a plain Value or a normal function that returns a Flow :D 

So suspend can be removed here